src/views/ajax.js

```diff
@@ -74,6 +74,8 @@ function rowPopupTemplate(request) {
 function update(request) {
     state.count++;
 
+    document.getElementById('glimpse-ajax-popup-list').style.display = 'table';
```

Comment:
Can we prefer adding a class/attr. instead? 🙏 e.g., `data-glimpse-visible="true"` or `glimpse-table-visible`, something like that.